.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -2,185 +2,193 @@ name: Bug Report
+              - "MSAL.js (@azure/msal-browser)"
+              - "MSAL.js v1 (@azure/msal or msal)"
+              - "MSAL Node (@azure/msal-node)"
+              - "Passport Azure AD (passport-azure-ad)"
+              - "ADAL Node (adal-node)"
+              - "ADAL.js (adal.js)"
+      validations:
+          required: true
+
+    - type: input
+      attributes:
+          label: Core Library Version
+          description: "Please enter the latest version this issue can be reproduced in."
+          placeholder: "3.1.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Wrapper Library
+          options:
+              - "MSAL Angular (@azure/msal-angular)"
+              - "MSAL React (@azure/msal-react)"
+              - "MSAL Node Extensions (@azure/msal-node-extensions)"
+              - "ADAL Angular (adal-angular)"
+              - "Not Applicable"
+      validations:
+          required: true
+
+    - type: input
+      attributes:
+          label: Wrapper Library Version
+          description: "Please enter the latest version this issue can be reproduced in. If you are not using a wrapper library please enter 'None'"
+          placeholder: "2.0.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Public or Confidential Client?
+          description: Are you using PublicClientApplication or ConfidentialClientApplication? (Issues regarding msal-browser, msal-react and msal-angular are always "Public")
+          multiple: true
+          options:
+              - "Public"
+              - "Confidential"
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Description
+          description: "Please briefly describe your issue."
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Error Message
+          description: "Please provide any error message you are receiving and a stack trace."
+
+    - type: textarea
+      attributes:
+          label: Msal Logs
+          description: "Please provide verbose level log messages from msal. Bug reports without verbose MSAL logs will be closed without comment."
+
+    - type: textarea
+      attributes:
+          label: Sanitized Network Trace
+          description: "Please upload a sanitized network trace (preferably from Fiddler, but network .HAR files are acceptable) containing the requests and responses from the failed authentication scenario. **Important: Please make sure to properly scrub all sensitive information from the trace, including but not limited to access/refresh tokens, client secrets, identifiable user information, etc.** Bug reports without sanitized network traces can be closed without comment. [Here's a guide](https://learn.microsoft.com/en-us/azure/azure-web-pubsub/howto-troubleshoot-network-trace) on how to collect a network trace."
```

Comment:
Let's make sure we have instructions on how to scrub traces before we start asking for this.

Reply:
Removed for now.